recipes/epysurv/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "epysurv" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  url: https://github.com/jarnorfb/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 56383ce024e2ad76c1ac300065efe555cc57fcbc8d7ab00052b310195470e80c
+
+build:
+  # noarch: python
+  number: 0
+  merge_build_host: True  # [win]
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+  host:
+    - python=3.7
+    - pip
+    - r-base=3.5
+    - rpy2
+    - numpy=1.16
+    - r-essentials
+  run:
+    - python=3.7
+    - libgfortran=3
+    - r-essentials
+    - rpy2
+    - numpy
+    - matplotlib
+    - pandas
+    - scipy
+    - seaborn
+    - plotnine
+
+test:
+  imports:
+    - epysurv
+    - epysurv.models
+    - epysurv.data
+    - epysurv.visualization
+    - epysurv.simulation
+    - epysurv.metrics
+
+about:
+  home: http://github.com/simplejson/simplejson
+
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+
+  description: |
+    epysurv provides access to epidemiological surveillance in Python by
+    wrapping the R surveillance package.
+  dev_url: https://github.com/JarnoRFB/epysurv
+
+extra:
+  recipe-maintainers:
+    - JarnoRFB
+    - justinshenk
```

Comment:
All of the maintainers need to comment that they are ok maintaining the package.

Reply:
Hi, I am ok with maintaining the package :)